src/scythe-bidder/create-room.tsx

```diff
@@ -73,72 +57,58 @@ export default function CreateRoom({ onCreate }: { onCreate: () => void }) {
               </a>
             </i>
           </p>
+
+          <p>Choose "Custom" to select and view exact configurations.</p>
         </div>
       ),
       onOk() {},
     });
   }
 
-  const onClick = React.useCallback(async () => {
-    const numPlayersNum = Number(numPlayers);
-    let setupData = null;
-    if (activeCombinations === GameSetting.BASE) {
-      setupData = {
-        factions: FACTIONS_BASE,
-        mats: MATS_BASE,
-        setting: "Base",
-      };
-    }
-    if (activeCombinations === GameSetting.IFA) {
-      setupData = {
-        factions: FACTIONS_IFA,
-        mats: MATS_IFA,
-        setting: "IFA",
-      };
-    }
-    if (activeCombinations === GameSetting.HI) {
-      setupData = {
-        factions: FACTIONS_HI,
-        mats: MATS_HI,
-        setting: "Hi-Tier",
-      };
-    }
-    if (activeCombinations === GameSetting.LO) {
-      setupData = {
-        factions: FACTIONS_LO,
-        mats: MATS_LO,
-        setting: "Lo-Tier",
-      };
-    }
-    // this if check should be unnecessary
-    if (
-      !numPlayersNum ||
-      numPlayersNum < MIN_PLAYERS ||
-      (activeCombinations !== GameSetting.IFA &&
-        numPlayersNum > MAX_PLAYERS_BASE) ||
-      (activeCombinations === GameSetting.IFA &&
-        numPlayersNum > MAX_PLAYERS_IFA)
-    ) {
-      return;
-    }
+  let factionsSet = new Set<Faction>();
+  let matsSet = new Set<Mat>();
+
+  Object.entries(combosMap).forEach(([f, matsObj]) => {
+    Object.entries(matsObj).forEach(([m, v]) => {
+      if (v) {
+        factionsSet.add(f as Faction);
+        matsSet.add(m as Mat);
+      }
+    });
+  });
+
+  const shouldDisableButton =
+    factionsSet.size < numPlayers || matsSet.size < numPlayers;
```

Comment:
Good thinking here^

Reply:
To be clear, it's still possible to submit an invalid configuration (select all mats with one faction, and select all factions with one mat, and try anything greater than 2 players). I updated the error message to hint that this might be the case, but it doesn't seem straightforward to surface the server-side error to the client.